docs/pages/guides/using-sentry.mdx

```diff
@@ -283,6 +169,39 @@ Once configured, information about the associated update will show up in an erro
 
 </Collapsible>
 
+## Sentry Integration with EAS
+
+The Sentry integration with Expo allows you to view crash reports and Session Replays for your Expo app deployments directly within your Expo Application Services (EAS) dashboard. This integration provides a direct link to Sentry stack traces with full context, session replays, and debugging capabilities.
+
+### Install
+
+> Sentry owner, manager, or admin permissions are required to install this integration.
+
+1. Log into EAS and open **Account Settings > Overview** in EAS (e.g., `https://expo.dev/accounts/[your-account]/settings`).
```

Comment:
```suggestion
1. Log in into you Expo account and open [**Account settings > Overview**](https://expo.dev/accounts/[your-account]/settings).
```

Let's update the verbiage for consistency and link the settings path so that already logged-in users can open the link directly